mintpy/view.py

```diff
@@ -1342,12 +1353,17 @@ def plot(self):
                 data[data != 0.] -= ref_data
 
             # masking
-            if self.zero_mask:
-                vprint('masking pixels with zero value')
-                data = np.ma.masked_where(data == 0., data)
             if self.msk is not None:
                 vprint('masking data')
                 data = np.ma.masked_where(self.msk == 0., data)
+            if self.msk is None:
+                self.msk = np.ones(data.shape, dtype=np.int8)
+            if self.zero_mask:
+                vprint('masking pixels with zero value')
+                self.msk = np.ma.masked_array(self.msk, mask=np.isnan(data))
+                np.ma.set_fill_value(self.msk, 0)
+                self.msk=self.msk.filled()
+                data = np.ma.masked_where(self.msk == 0., data)
```

Comment:
This modification does not work as expected, for the Fernandina example dataset: the water body, which has zero velocity, should be masked out.

```
view.py velocity.h5 velocity --mask no --zero-mask
```

<img width="1112" alt="Screen Shot 2020-03-10 at 3 17 56 PM" src="https://user-images.githubusercontent.com/13143710/76364462-713a5280-62e2-11ea-9242-c371cc712bcc.png">